sagemaker-clarify/online_explainability/natural_language_processing/code/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers==4.2.1
```

Comment:
Is this file being used?

Reply:
yes, all files in code directory are being packaged in model container.

Reply:
it looks like its saying pip install sagemaker captum --upgrade and then it doesn't know what captum is, I believe the fix here is to get rid of sagemaker and just make it !pip install captum --upgrade?

Reply:
It could be, though when manually running the notebook it's not causing an issue. I can try that once.

Reply:
We're currently testing this on our end